src/main/java/com/cloudbees/jenkins/plugins/advisor/utils/EmailUtil.java

```diff
@@ -17,9 +23,15 @@ public static String fixEmptyAndTrimAllSpaces(String value) {
   }
 
   public static String urlEncode(String value) {
-    if (value != null) {
-      value = value.replaceAll(",", "%2C");
+    if (value == null) {
+      return null;
+    }
+    try {
+      return URLEncoder.encode(value, "UTF-8");
+    } catch (UnsupportedEncodingException unsupportedEncodingException) {
+      LOG.warning(
+        "Unable to URLEncode with UTF-8 the value " + value + " : " + unsupportedEncodingException.getMessage());
```

Comment:
nit: The exception is for the provided encoding, ie in our case it will never trigger.

Reply:
yes not sure what to put in term of message (but I agree that it should never happen)

Reply:
better like this @PierreBtz ?